.github/actions/markPullRequestsAsDeployed/markPullRequestsAsDeployed.js

```diff
@@ -1,13 +1,16 @@
+const _ = require('underscore');
 const core = require('@actions/core');
 const {context} = require('@actions/github');
+const moment = require('moment');
 const ActionUtils = require('../../libs/ActionUtils');
 const GithubUtils = require('../../libs/GithubUtils');
 
 
 const prList = ActionUtils.getJSONInput('PR_LIST', {required: true});
 const isProd = ActionUtils.getJSONInput('IS_PRODUCTION_DEPLOY', {required: true});
-const isCP = ActionUtils.getJSONInput('IS_CHERRY_PICK', {required: false}, false);
+const stagingDeployIssueNumber = ActionUtils.getJSONInput('STAGING_DEPLOY_NUMBER', {required: true});
```

Comment:
You'll need to add this new input to the [action metadata file](https://github.com/Expensify/Expensify.cash/blob/f38f41ef96ec94c77d9ac423263271257a99b0b6/.github/actions/markPullRequestsAsDeployed/action.yml)